src/renderer/views/Playlist/Playlist.js

```diff
@@ -241,6 +241,20 @@ export default defineComponent({
     sortBySelectValues() {
       return this.sortByValues
     },
+    sortIcon: function () {
+      switch (this.sortOrder) {
+        case SORT_BY_VALUES.DateAddedNewest:
```

Comment:
Maybe use numeric icon for date related options?

Reply:
I don't think that's the right choice for dates, but I am open to being convinced

Reply:
OK seems too small to be read if used
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/3a9bdede-c7d8-44bc-bba0-a5b455c30e0f)